packages/icons/assets/Code.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="10" viewBox="0 0 16 10" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M4.99999 9.70799L0.291992 4.99999L4.99999 0.291992L5.77099 1.06199L1.83299 4.99999L5.77099 8.93799L4.99999 9.70799ZM11 9.70799L10.229 8.93799L14.167 4.99999L10.229 1.06199L11 0.291992L15.708 4.99999L11 9.70799Z" fill="#F3F6F8" fill-opacity="0.6"/>
```

Comment:
```suggestion
<path d="M4.99999 9.70799L0.291992 4.99999L4.99999 0.291992L5.77099 1.06199L1.83299 4.99999L5.77099 8.93799L4.99999 9.70799ZM11 9.70799L10.229 8.93799L14.167 4.99999L10.229 1.06199L11 0.291992L15.708 4.99999L11 9.70799Z" fill="currentColor" />
```

Gotta change the fill colors of svgs when we add them :)

Reply:
oops!